commit f2f04940883385893fd120854bea0e14f9c218e2 (refs/changes/68/368/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-08-29T12:34:19-07:00 (3 years, 1 month ago)
    
    trentm/play#6 add a "repository" key to package.json so npm stops complaining

diff --git a/README.md b/README.md
index 260f359..dc45115 100644
--- a/README.md
+++ b/README.md
@@ -5,3 +5,5 @@ playin' around. nothing formal
 ## How do merges work now?
 
 Let me count the ways...
+1
+2
diff --git a/package.json b/package.json
index 61d7d56..c75d173 100644
--- a/package.json
+++ b/package.json
@@ -4,5 +4,9 @@
   "description": "",
   "main": "lib/index.js",
   "author": "trentm@gmail.com",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/trentm/play.git"
+  },
   "license": "MIT"
 }
